ublox/src/ubx_packets/packets.rs

```diff
@@ -2135,6 +2140,38 @@ bitflags! {
 //     msss: u32,
 // }
 
+#[ubx_packet_recv]
+#[ubx(class = 0x02, id = 0x13, max_payload_len = 72)]
+struct RxmSfrbx {
+    gnss_id: u8,
+    sv_id: u8,
+    reserved1: u8,
+    freq_id: u8,
+    num_words: u8,
+    reserved2: u8,
+    version: u8,
+    reserved3: u8,
+
+    #[ubx(map_type = Vec<u32>, from = vec_conversion::convert_to_u32_vec)]
+    dwrd: [u8; 0],
+}
+
+mod vec_conversion {
+    use std::convert::TryInto;
+
+    pub(crate) fn convert_to_u32_vec(bytes: &[u8]) -> Vec<u32> {
```

Comment:
Something like this @notoriaga @john-michaelburke maybe better instead of having an iterator?
Think we can also do the same for `MonVerExtensionIter` since `dwrd` looks to be fixed with max payload length, we just convert this into a vec.

Reply:
Maybe there's a better way to implement `From<&[u8]> for [u32]` or something instead of manually writing that

Reply:
Could maybe re-use `FixedLinearBuffer`

Reply:
I wonder if you could use something like chunks and possibly a map to u32 values.
https://doc.rust-lang.org/std/slice/struct.Chunks.html

Reply:
I think the iterator is better here, it requires no allocation unlike the vec approach. that's probably why similar fields in this crate use an iterator. And should you want the items in a vec you can just `msg.field().collect::<Vec<_>>()` it (or you could collect into a different data structure)

Reply:
And I agree I think chunks would be nice here (or maybe chunks exact? not sure how we are supposed to handle extra bytes. Could do something like this (assuming that the offset is just for making sure you have enough bytes for a u32?)

```rust
struct DwrdIter<'a> {
    inner: core::slice::ChunksExact<'a, u8>,
}

impl<'a> DwrdIter<'a> {
    fn new(bytes: &'a [u8]) -> Self {
        Self {
            inner: bytes.chunks_exact(4),
        }
    }
}

impl<'a> Iterator for DwrdIter<'a> {
    type Item = u32;

    fn next(&mut self) -> Option<Self::Item> {
        let chunk = self.inner.next()?;
        let n = u32::from_ne_bytes(chunk.try_into().unwrap());
        Some(n)
    }
}
```